.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install packages
+      run: |
+        sudo apt-get -y install pandoc
+      if: matrix.python-version == 3.6
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
```

Comment:
I guess these `uses` are basically means calling a script, in this case the script is `actions/setup-python` at version `v2?

Reply:
That's exactly it.